roles/dnsmasq/templates/dnsmasq-ds.yml

```diff
@@ -25,7 +25,7 @@ spec:
             capabilities:
               add:
                 - NET_ADMIN
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
`Always` make sense when you have a moving image tag (canary/latest)
here with `image: dnsmasq:2.72`  a IfNotPresent is perfectly fine